tests/testsuite/workspaces.rs

```diff
@@ -698,6 +698,7 @@ fn share_dependencies() {
 [UPDATING] `dummy-registry` index
 [LOCKING] 1 package to latest compatible version
 [ADDING] dep1 v0.1.3 (available: v0.1.8)
+[NOTE] 1 package may have a higher, compatible versions. To update it, run `cargo update <name> --precise <version>
```

Comment:
This is an example of where it might show up for a non-MSRV case (and the suggested action will fail)